<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="mStackedWidget">
     <widget class="PreviewWidget" name="mPreview"/>
     <widget class="PageWidget" name="mBook"/>
     <widget class="ConfigWidget" name="mConfig"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageWidget</class>
   <extends>QWidget</extends>
   <header>pagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigWidget</class>
   <extends>QWidget</extends>
   <header>configwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
